src/libcore/ops.rs

```diff
@@ -171,6 +179,44 @@ use marker::Unsize;
 ///     let _x = HasDrop;
 /// }
 /// ```
+///
+/// Showing the recursive nature of `Drop`. When `outer` goes out of scope, the
```

Comment:
Only as much as the first sentence to the previous example which says "A trivial implementation of `Drop`".